packages/mongodb-chatbot-server/src/app.ts

```diff
@@ -45,15 +44,16 @@ export interface AppConfig {
   apiPrefix?: string;
 
   /**
-    Whether to serve a static site from the root path (`GET https://my-site.com/`).
-    Defaults to false.
-    This is useful for demo and testing purposes.
-
-    You should probably not include this in your production server.
-    You can control including this in dev/test/staging but not production
-    with an environment variable.
+    Additional server logic using the server's `Express` app object.
+    Can do things like add additional, routes, global middleware, etc.
+    Runs immediately after the app is instantiated,
+    before additional routes and middleware are added.
+    @example
+    (app)=> {
+      app.get("/", (req, res) => res.send({ hello: "world" }))
+    }
    */
-  serveStaticSite?: boolean;
+  additionalServerLogic?: (app: Express) => Promise<void>;
```

Comment:
```suggestion
  appSetup?: (app: Express) => Promise<void>;
```

nit: could we rename to be a bit more specific?